Frends.PowerShell.RunScript/Frends.PowerShell.RunScript/Frends.PowerShell.RunScript.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<TargetFramework>net471</TargetFramework>
```

Comment:
Target framework should be net6.0

Reply:
I was going to do the target framework change after the initial implementation. Task does not automatically work in net6.0.

Reply:
Target framework changed to only include .NET 6.0